app/controllers/foreman_gutterball/api/v2/content_reports_controller.rb

```diff
@@ -1,17 +1,43 @@
 module ForemanGutterball
   module Api
     module V2
-      class ContentReportsController < Katello::Api::V2::ApiController
-        def index
-          # do stuff
+      class ContentReportsController < ::Katello::Api::V2::ApiController
+        api :GET, '/reports', 'List available reports'
+        def reports
+          available_reports
         end
 
-        def show
-          # do other stuff
+        api :GET, '/reports/consumer_status', 'Consumer status reports'
+        def consumer_status_reports
+          consumer_status
         end
 
-        def run
-          # even more stuff
+        api :GET, '/reports/consumer_trend', 'Consumer trend reports'
+        def consumer_trend_reports
+          consumer_trend
+        end
+
+        api :GET, '/reports/status_trend', 'Status trend reports'
+        def status_trend_reports
+          status_trend
+        end
+
+        private
+
+        def available_reports
+          GutterballService.new.available_reports
+        end
+
+        def consumer_status
+          GutterballService.new.consumer_status
+        end
+
+        def consumer_trend
+          GutterballService.new.consumer_trend
+        end
+
+        def status_trend
+          GutterballService.new.status_trend
```

Comment:
it might make a little more sense to define a method that returns a new `GutterballService` so that you aren't visually calling `status_trend` (and others) twice.

like this:
```ruby
def service
  GutterballService.new
end
```

then each of your actions can be shortened some:
```ruby
def reports
  service.consumer_status
end

def consumer_status_reports
  service.consumer_status
end

def consumer_trend_reports
  service.consumer_trend
end

def status_trend_reports
  service.status_trend
end
```

and you can get rid of all those intermediary methods